### PR TITLE
Fix tools local ec

### PR DESCRIPTION
**Pull request by wprzechrzta**

### CODE REVIEW

**Review by Automaat — DISMISSED** [review]

File: tools/docker-compose.yaml [hunk 1]
```diff
@@ -3,13 +3,13 @@ version: '3'
 services:
   consul:
     container_name: consul
-    image: consul:latest
+    image: consul:1.5.3
```
Comment (lukidzi) [inline tools/docker-compose.yaml:6]: Let's keep latest here.

File: tools/envoy/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM allegro/envoy-dev:v1.16.1-dev-lua-segfault-fix-1-16-0-backport-20201118-df9dc819
+FROM envoyproxy/envoy-alpine:v1.17.1
```
Comment (lukidzi) [inline tools/envoy/Dockerfile:1]: Let's keep in sync with https://github.com/allegro/envoy-control/blob/master/envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt#L37

File: tools/docker-compose.yaml (hunk 1 above)
Comment (wprzechrzta) [reply tools/docker-compose.yaml:6]: I would stick to this version as this is our current target, with latest there is no control what is actually used

Comment (slonka) [reply tools/docker-compose.yaml:6]: I'm for repeatable builds as well. So I would pick a version and stick with it.

**Comment by slonka** [comment]
@Kornelos you should check out this PR. I think what happened in your case was that we did not rebuild the containers (see `--build --nodeps` in the command) and you were simply running the old version.

File: tools/envoy/Dockerfile (hunk 2 above)
Comment (wprzechrzta) [reply tools/envoy/Dockerfile:1]: updating here would require bigger investment, after updating and customization in 
package manager https://github.com/allegro/envoy-control/blob/master/envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt#L22  to RUN apk add --no-cache curl iproute2 iptables dnsmasq i couldn't make it working as we have custom setup

**Review by lukidzi — APPROVED** [review]

**Review by Automaat — APPROVED** [review]

**Review by slonka — APPROVED** [review]

**Pull request merged**